### DISCUSSION TITLE
打开b站链接时报错

### DISCUSSION THREAD

**Discussion by Asukaaaaaa** [hooke007/mpv_PlayKit#472] [mpv 通用]
命令行打开视频
```
mpv https://www.bilibili.com/video/BV1rZ4y1879b
```
![image](https://github.com/hooke007/MPV_lazy/assets/24503369/3d337fbf-0abd-4010-a415-a77d22c63620)
貌似是解码相关的问题，有没有大佬遇到过
单独用yt-dlp下载视频没问题
用浏览器插件获取视频下载链接再用mpv直接打开这个链接播放也没问题
不知道是哪一个环节出了bug😥

**Comment by hooke007** [comment]
ffmpeg的问题先自行更新mpv
我这里较新的mpv没有复现
